docs/Configuration.md

```diff
@@ -402,6 +405,23 @@ The complete list of modes are:
 
     _Prerequisite:_ Park-and-ride areas near the station need to be present in the OSM input file.
 
+- `CAR_RENT`: Taking a rented, shared-mobility car for part or the entirety of the route.
+
+    _Prerequisite:_ Vehicle positions need to be added to OTP  as dynamic data feeds.
+
+    For dynamic car positions configure an input feed. See [Configuring real-time updaters](Configuration.md#configuring-real-time-updaters).
+
+- `CAR_HAIL`: Hailing a car from a transportation network company such as Lyft or Uber and then riding in the vehicle for part or the entirety of the route. OTP will still use the underlying graph to calculate driving directions, so results will be different travel time estiamtes quoted from the TNC providers. If service is not available at a particular pickup location, this can cause a TNC availability error which can result in an itinerary not being able to be calculated.
+
```

Comment:
`OTP will still use the underlying graph to calculate driving directions, so results will be different travel time estiamtes quoted from the TNC providers.` --> `OTP will still use its underlying graph to calculate driving directions, so car leg results will likely contain different travel time estimates than those quoted from the TNC providers.`